File: jscomp/test/minimal_test.js
'use strict';


fake_y = --[ :: ]--[
  2,
  --[ :: ]--[
    3,
    --[ [] ]--0
  ]
];

fake_z = --[ :: ]--[
  1,
  fake_y
];

exports.fake_y = fake_y;
exports.fake_z = fake_z;
--[ No side effect ]--

File: jscomp/test/minimal_test.lua
__console = {log = print};


fake_y = --[[ :: ]]{
  2,
  --[[ :: ]]{
    3,
    --[[ [] ]]0
  }
};

fake_z = --[[ :: ]]{
  1,
  fake_y
};

exports = {};
exports.fake_y = fake_y;
exports.fake_z = fake_z;
return exports;
--[[ No side effect ]]
